src/native/libs/System.Native/CMakeLists.txt

```diff
@@ -50,6 +50,11 @@ else()
     list (APPEND NATIVE_SOURCES pal_environment.c)
 endif()
 
+if (CLR_CMAKE_TARGET_MACCATALYST OR CLR_CMAKE_TARGET_IOS OR CLR_CMAKE_TARGET_TVOS)
```

Comment:
Is it better to just merge this block with the block above and below as they all have the same condition of `CLR_CMAKE_TARGET_MACCATALYST OR CLR_CMAKE_TARGET_IOS OR CLR_CMAKE_TARGET_TVOS`?